### PR TITLE
PDA-241 add multi uischema process definition

### PR DESCRIPTION
**Pull request by rimselis**
branched from PDA-240 #118

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ffleandro**

If they are not already assigned, you can assign the PR to them by writing `/assign @ffleandro` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js
Comment (raxkaynan) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js:56]: I'm not sure if it's necessary to `JSON.parse()` child properties here as the entire object has already been parsed. Am I missing something here?

**Review by raxkaynan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/119#pullrequestreview-410829483" title="Approved">raxkaynan</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ffleandro**

If they are not already assigned, you can assign the PR to them by writing `/assign @ffleandro` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
